CHANGELOG.md

```diff
@@ -1,5 +1,45 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.15.1
+
+<!-- generated comparing v5.15.0..master -->
+
+_Dec 18, 2023_
+
+A big thanks to the 13 contributors who made this release possible.
+This release was mostly about 🐛 bug fixes and 📚 documentation improvements.
+
+### `@mui/material@5.15.1`
+
+- &#8203;<!-- 13 -->[material-ui][CardActions][DialogActions] Apply margin for all children except for 1st child (#40168) @sai6855
```

Comment:
We can remove the [material-ui] tag (and others in sections below) from PR descriptions